edv/test_scripts/client/testlists/testlist_efispec.sh

```diff
@@ -1,66 +1,58 @@
 #!/bin/sh
 
+# Get the common routines to parse command line
+source ${RUNDIR}/testlists/testlist_common.sh
+
+# Get the global environment variables
+source ${RUNDIR}/scripts/client_env.sh
+
+# Parse the command line
+parseAndSetParameters $@
+
 export SCM="700G" #700G
 export NVME="30T" #1T
 
 export SRV_HEADNODE="edaos09"
 export MPI="IMPI"
-export RUNDIR="/panfs/users/schan15/client"
-export BUILDDIR="/panfs/users/schan15/builds"
-export TB="daos_rel201rc1"
+export RUNDIR="/panfs/users/${USER}/client"
+export BUILDDIR="/panfs/users/${USER}/builds"
 export MOUNTDIR="/tmp/daos"
-export APPSRC="/panfs/users/schan15/apps/${MPI}/efi_johann"
+export APPSRC="/panfs/users/${USER}/apps/${MPI}/efi_johann"
```

Comment:
Why are we having efi_johann static directory? Any reason?

Reply:
The directory 'efi_johann' is the directory that we store EFISPEC in. We use a version of EFISPEC that Johann modified and tarred up. When you untar the file you get the directory efi_johann. 

I agree with both your comments that we should make the name/paths to applications a little more flexible in our scripts or include the applications in the repo.

Reply:
Maybe, save Johann's tar file in the git repo.